lib/helpers/sassNodeModulesAliasImporter.js

```diff
@@ -0,0 +1,49 @@
+var path = require("path");
+var findNodeModules = require("find-node-modules");
+var fs = require("fs");
+
+var NODE_MODULES_ALIAS = "~";
+var sassFileExtensions = [".sass", ".scss"];
+
+module.exports = function sassNodeModulesAliasImporter(config) {
+	if (typeof config.sassNodeModulesAlias === "string") {
+		NODE_MODULES_ALIAS = config.sassNodeModulesAlias;
+	}
+
+	return function sassNodeModulesAliasImporter(url, prev) {
+		if (url.substr(0, NODE_MODULES_ALIAS.length) === NODE_MODULES_ALIAS) {
+			var i, j, k;
+			var paths = findNodeModules({ cwd: prev, relative: false }).map(function(
```

Comment:
By convention `node_modules` are located in the folder where `mango.yaml` lives. Can't we just use that location?

Reply:
🤷‍♂️ probably. Thought this would be more correct. Monorepo workspace comes to mind, where this would work better.